src/gui/qt/gui.cpp

```diff
@@ -27,197 +27,286 @@ const int yOffset = -380;
 
 // Inspired by CEmu's approach to key events
 class KeyboardEventFilter : public QObject {
-    Q_OBJECT
+  Q_OBJECT
 
 public:
-    bool eventFilter(QObject* obj, QEvent* event) override;
```

Comment:
Are all these formatting changes on purpose?

Reply:
sorry no, just realized after pushing that my editor made strange formatting stuff.
Of course you can scrap all these dummy proposal for formatting changes.
Sorry about that.